schemas/constructs/component/v1beta1/component.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$id": "https://schemas.meshery.io/component.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Components are the atomic units for designing infrastructure. Learn more at https://docs.meshery.io/concepts/components",
+  "additionalProperties": false,
+  "type": "object",
+  "properties": {
+    "schemaVersion": {
+      "type": "string",
+      "description": "Specifies the version of the schema used for the definition."
+    },
+    "definitionVersion": {
+      "type": "string",
+      "description": "Specifies the version of the definition."
+    },
+    "displayName": {
+      "$ref": "https://schemas.meshery.io/core.json#inputString",
+      "description": "Name of the component in human-readible format."
+    },
+    "description": {
+      "type": "string",
+      "description": "A written representation of the purpose and characteristics of the component."
+    },
+    "format": {
```

Comment:
There's an argument for moving this `format` field into the `component` stanza (section).